.github/workflows/elm_tests.yml

```diff
@@ -119,11 +122,26 @@ jobs:
           ./bin/check_website-copy_paths.sh website-copy
 
   smoke-test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     steps:
       - name: Checkout code
         uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791
 
       - name: Run Smoke Tests in Docker
         run: bin/run-tests-in-docker.sh
+
+  test-elm-repo-solutions:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout elm-analyzer
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
```

Comment:
I would suggest building the Docker image here in CI, which will allow it to be cached between run. 
See https://github.com/exercism/generic-representer/blob/main/.github/workflows/ci.yml#L19-L32 for an example (note: the image name must be the same here and in the analyze-all-exercises-in-docker.sh script)

Reply:
Thank you, I'll take a look.
But does it really make sense to cache it when the source code will most likely change in most PRs?

Reply:
Well, you'd still get to use the Docker cache for these lines: https://github.com/exercism/elm-analyzer/blob/main/Dockerfile#L1-L28

So I think it would still be useful

Reply:
@ErikSchierboom I added the caching, can you have a look?

Reply:
LGTM!